index.html

```diff
@@ -3,12 +3,762 @@
+                a concept known as the multiverse.</p>
+            <p>So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper
+                into the past.</p>
+            <p>We might never know what happened before inflation, at the very beginning, because inflation erases
+                everything that came before it. All the chaos and randomness of the primordial moment are swept away,
+                forever out of our view.</p>
+            <p>“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of
+                the team, “you wind up at inflation.”</p>
+
+            <div class="order">
+                <p>A version of this article appears in print on March 18, 2014, Section A, Page 1 of the New York
+                    edition with the headline: Space Ripples Reveal Big Bang’s Smoking Gun. <span><a href="#">Order
+                            Reprints</a> | <a href="#">Today’s Paper</a> | <a href="#">Subscribe</a></span></p>
+            </div>
+
+            <div class="social-bottom">
+                <div><button class="end-social-btn">READ 615 COMMENTS</button></div>
+                <ul class="date-social-icons">
+                    <li><a href=""><img src="images/facebook.svg" class="social-btn" alt="facebook icon" /></a></li>
+                    <li><a href=""><img src="images/twitter.svg" class="social-btn" alt="twitter icon" /></a></li>
+                    <li><a href=""><img src="images/envelop.svg" class="social-btn" alt="mail icon" /></a></li>
+                    <li><a href=""><img src="images/next-arrow.svg" class="social-btn" alt="next arrow icon" /></a></li>
+
+                </ul>
+            </div>
+
+        </div>
+
+
+
+    </section>
+
+    <!-- Format This Section Properly -->
+
+    <div class="pre-footer">
+        <div class="popular-space">
+            <h1>More in space and Astronomy</h1>
+            <div class="space">
+                <div class="ppl-item">
+                    <img src="https://static01.nyt.com/images/2020/05/08/science/00COMET/00COMET-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale"
+                        alt="item1" />
+                    <h3>Damian Peach, Chilescope</h3>
+                    <h2>How to See Comet SWAN in Night Skies</h2>
+                    <p>May 12</p>
+                </div>
+                <div class="ppl-item">
+                    <img src="https://static01.nyt.com/images/2020/05/06/science/06SCI-STARLINK/06SCI-STARLINK-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale"
+                        alt="item2" />
+                    <h3>Peter Komka/EPA, via Shutterstock</h3>
+                    <h2>
+                        SpaceX Plans Sunshades to Save Night Skies From Starlink
+                        Satellites
```

Comment:
- [ ] Please make the text (just title) and images links, just like the original site.

Reply:
Thanks for the review, the links have been added to the images and text as requested. Your comment on making descriptive commits have been noted and utilized in the changes made. Bravo!!!